src/ClusterBootstrap/services/monitor/pre-render.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+dir=`dirname $0`
+
+file_name=${dir}/grafana-config.yaml
+
+# create configmap
+for i in `find ${dir}/grafana-config/ -type f -regex ".*json" ` ; do
+    echo --from-file=$i
+done | xargs ${dir}/../../deploy/bin/kubectl --namespace=kube-system create configmap grafana-configuration --dry-run -o yaml >> $file_name
```

Comment:
call kubectl directly doesn't work. please use ./deploy.py kubectl ***

Reply:
This is used to generate configmap yaml file through files, it doesn't need to be called via `deploy.py kubectl`.